### DISCUSSION TITLE
Gotec A 2-digit error code E34 - No entries to navigate

### DISCUSSION THREAD

**Discussion by ivankrajnc** [keirf/flashfloppy#784] [General]
Hello everyone.

I own the Korg Triton Rack version.
I recently bought a Gotec SFR1M44-U100K, since the Rack version only has a regular floppy drive.  Board SFRKC30.AT2, microcontroller is AT32F415 (KBU7-4 5FR4G-000 2133CT), previous disk drive was Panasonic JU-256A198P.
I ran into a problem where the emulator reads only the first disk when reading, but then it doesn't find the second one....

I flashed the AT32F415 MCU as instructed: flashfloppy-at415-st105-3.38.hex
Now I'm getting the E34 error with various FAT32 formatted USB sticks.
Before flashing, Gotek normally read all USBs, where it now shows error E32.

Does anyone know a solution?

Thanks in advance :)

**Comment by keirf** [comment]
FlashFloppy works with IMG files on a single FAT32 filesystem on your USB stick. You don't partition it with partitionmanager, as with factory firmware.

**Reply by ivankrajnc** [reply]
Thanks for the quick reply.
I also tried the normal formatting to FAT32, with data and without data. In all cases, Gotek adds an empty IMAGE_A.CFG file to the USB.
Does it matter if FAT32 is formatted to 4096B, 8192B, 16KB, 32KB or 64KB?

**Reply by keirf** [reply]
IMAGE_A.CFG will contain path to the last loaded image file.

It shouldn't matter about FAT32 cluster size. But you will need a valid IMG file, or other sort of supported image file type.

**Reply by ivankrajnc** [reply]
Thank you. Now I'm a step further and I created the IMG file and I get the E31 error.
Are there or are there any rules regarding the composition, size, etc. of the IMG file written so that Gotek can recognize them?

**Reply by keirf** [reply]
Yes. You need a text file called FF.CFG on your USB stick containing the line `host = akai`

**Reply by ivankrajnc** [reply]
Apparently there is something wrong with my IMG.
In the meantime, at https://github.com/keirf/flashfloppy-images/blob/master/README.md I found an unformatted file 1m44.img, Gotek reads this file, but mine does not. I am attaching the files that are on the USB.
[E31.zip](https://github.com/keirf/flashfloppy/files/11355761/E31.zip)

**Comment by keirf** [comment]
Your problem is that it's a CD IMG, not a floppy IMG. I don't know if Korg Triton accepts FAT filesystem anyway? You may need an IMG of the correct size and then formatted on the Korg Triton?

**Reply by ivankrajnc** [reply]
Thanks for your time and advice, I've come further now, but still not up to reading and writing.

I have now created a floppy IMG with a size of 1.44MB and 512 Bytes/Block, as supported by Triton Rack (see attached image).
Gotek recognized the USB and alternately displays "0_" and "000".

Triton does not recognize the data and displays "no disk"...

I am attaching files.
[Korg Triton Rack.zip](https://github.com/keirf/flashfloppy/files/11359456/Korg.Triton.Rack.zip)

**Comment by ivankrajnc** [comment]
Today I tried modifying the FF.CFG Configuration File
I tried in order:
- interface = sugarart
- interface = ibmpc
- interface = ibmpc-hdout --> it recognized the USB and wrote "Unformatted" :)

Following your advice I formatted it on Triton and........THE THING WORKS!

Thank you very much for all the advice, have a nice day!

Ivan
